apps/fyler/src/aws_cli.erl

```diff
@@ -22,6 +22,15 @@ copy_folder(From,To) ->
 copy_folder(From,To,Acl) ->
   os:cmd(io_lib:format("aws s3 sync --acl ~s ~s ~s",[access_to_acl(Acl),From,To])).
```

Comment:
Line 23 is too long:   os:cmd(io_lib:format("aws s3 sync --acl ~s ~s ~s",[access_to_acl(Acl),From,To]))..